packages/expo-camera/src/CameraModule/barcode/BarCodeScannerWorker.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-env worker */
+
+declare var jsQR: Function;
+
+export default () => {
+  // @ts-ignore
+  self.importScripts('https://unpkg.com/jsqr@1.2.0/dist/jsQR.js');
```

Comment:
What's our approach towards importing libs from outside like here? Do we want sth like this or rather we would like to ship them ourselves? 🤔

Reply:
Not sure, I'll need to do more research to figure out how to make this work in a concise manner.